sdk/batch/Microsoft.Azure.Management.Batch/src/Generated/Models/BatchAccountIdentityUserAssignedIdentitiesValue.cs

```diff
@@ -1,60 +0,0 @@
-// <auto-generated>
```

Comment:
same breaking sdk change as above

Reply:
It looks like these two types are functionally equivalent.  Can we simply add this type back in as extending the new 'UserAssignedIdentities' type to avoid the breaking change?

Reply:
Still a breaking change, this class is still removed with no mitigation

Reply:
Added manually here: sdk/batch/Microsoft.Azure.Management.Batch/src/Models/BatchAccountIdentityUserAssignedIdentitiesValue.cs